base_search_mail_content/readme/ROADMAP.rst

```diff
@@ -2,3 +2,5 @@ This module restricts the message_content search functionality
 to internal users only, addressing the issue faced by project collaborators (portal users)
 as described in https://github.com/OCA/social/issues/1204. Consequently, portal users no
 longer have the ability to search within mail content.
+
+This module does not work correctly when using the GIN index with Japanese characters.
```

Comment:
Sorry it took me a while to check. From what I understand, the issue is related to how the trigram mechanism works and is not directly caused by the GIN index itself.

```suggestion
This module's usability is severely limited for languages that don't separate words with spaces (e.g., Chinese, Japanese, Korean, Thai, etc.), because `pg_trgm` needs three-character tokens and a similarity score above the default cutoff.
```